pkg/client/linstor.go

```diff
@@ -315,7 +308,16 @@ func (s *Linstor) Delete(ctx context.Context, vol *volume.Info) error {
 	}
 
 	// No snapshots, remove the resource.
-	return nil404(s.client.ResourceDefinitions.Delete(ctx, vol.ID))
+
+	// try to rm the RG, but don't consider it a failure if we can not (maybe other resources,...)
+	rd, rdErr := s.client.ResourceDefinitions.Get(ctx, vol.ID)
+	retErr := nil404(s.client.ResourceDefinitions.Delete(ctx, vol.ID))
+
+	if rdErr == nil && rd.ResourceGroupName != "" {
+		_ = s.client.ResourceGroups.Delete(ctx, rd.ResourceGroupName)
```

Comment:
Do we really want to suppress all errors?

Also, won't linstor generate an error report whenever this fails due to other resources being present?

Reply:
I thought it saves a query/check and in the worst case it is a RG hanging around. But generating an error report is a very valid point. I will update this part.